src/coverlet.core/CoverageSummary.cs

```diff
@@ -7,17 +7,17 @@
 
 namespace Coverlet.Core
 {
-    internal class CoverageSummary
+    internal static class CoverageSummary
```

Comment:
Same remark as for the eqt logger. All members could be put `static` so I have updated everything.

Reply:
the same